sim/scripts/scraper.py

```diff
@@ -26,9 +26,10 @@
     "Head Type": "head_type",
     "Drive Style": "drive_style",
     "Head Diameter": "head_diameter",
+    "Material": "material",
```

Comment:
Do the keys for these attributes need to be updated to the new schema too? `thread_pitch` is now `pitch` in new schema for example.

Reply:
Good catch!